styles.css

```diff
@@ -404,49 +404,61 @@ button:hover {
 
 .contact-part {
   width: 375px;
-  max-height: 970px;
-  height: auto;
-  padding: 100px 24px 0 24px;
-  background: #6070ff;
-  border-top-left-radius: 50px;
+  height: 690px;
```

Comment:
- [ Optional ] Please, avoid using fixed width and height because it affects the responsiveness of your website. I highly recommend you apply these changes even if I am making the as optional.